.github/template_files/flex_deploy.yaml

```diff
@@ -0,0 +1,214 @@
+name: Deploy Flex
+
+on:
+  # Run on merge to main
+  #push:
+  #  branches:
+  #  - main
+
+  # Enable running this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        type: environment
+        description: 'Environment to run tests against'
+      deploy_terraform:
+        required: true
+        type: boolean
+        default: true
```

Comment:
Let's default this to off for now until we can make the terraform script respect `overwrite_config`.